### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by MykolaFedasiuk**

### CODE REVIEW

**Comment by TarasHorin** [comment]
Good Job!)

**Comment by YuriyDmuhail** [comment]
Good job

**Review by JuliaStadnyk — APPROVED** [review]
Everything seems fine. Good job!

File: cypress/e2e/signIn.cy.js [hunk 1]
```diff
@@ -2,6 +2,15 @@
...
+   const email = 'pecyjynuq@mailinator.com'
+   const password = 'Password12345'
+
+   cy.visit('https://react-redux.realworld.io/') 
+   cy.contains('Sign in').click();
+   cy.get('[placeholder="Email"]').type(email)
+   cy.get('[placeholder="Password"]').type(password)
+   cy.contains('.btn', 'Sign in').click()
+   cy.contains('.navbar', 'Minimavoluptatumes')
+
```
Comment (tnboyko) [inline cypress/e2e/signIn.cy.js:14]:
```suggestion
   const email = 'pecyjynuq@mailinator.com';
   const password = 'Password12345';

   cy.visit('https://react-redux.realworld.io/');
   cy.contains('Sign in').click();
   cy.get('[placeholder="Email"]').type(email);
   cy.get('[placeholder="Password"]').type(password);
   cy.contains('.btn', 'Sign in').click();
   cy.contains('.navbar', 'Minimavoluptatumes');
```

File: cypress/e2e/signIn.cy.js [hunk 2]
```diff
@@ -2,6 +2,15 @@
...
-    
+   const email = 'pecyjynuq@mailinator.com'
+   const password = 'Password12345'
+
+   cy.visit('https://react-redux.realworld.io/') 
+   cy.contains('Sign in').click();
+   cy.get('[placeholder="Email"]').type(email)
+   cy.get('[placeholder="Password"]').type(password)
+   cy.contains('.btn', 'Sign in').click()
+   cy.contains('.navbar', 'Minimavoluptatumes')
```
Comment (tnboyko) [inline cypress/e2e/signIn.cy.js:13]: `cy.contains` is not an assertion, use `should` to check smth

**Review by tnboyko — APPROVED** [review]

**Pull request closed without merging**